OpenEdXMobile/res/layout/reset_dialog_email_input.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:baselineAligned="false"
```

Comment:
Here's what Android studio says about adding base alignment:

<img width="987" alt="screen shot 2016-12-15 at 5 19 45 pm" src="https://cloud.githubusercontent.com/assets/1710804/21223748/b7fd46e6-c2ea-11e6-8b76-2dba091727bc.png">

p.s. I don't think there'a need for now to have the `center_vertical` gravity since the height of the layout is `wrap_content`.